.estlintrc.ts

```diff
@@ -23,7 +25,7 @@ export default {
     {
       files: ["*.graphql"],
       parser: "@graphql-eslint/eslint-plugin",
-      plugins: ["@graphql-eslint"],
+      plugins: ["@graphql-eslint", "jsx-a11y"],
```

Comment:
can you issue a PR that includes this plugin @ https://github.com/LaunchWare/eslint-config-react/blob/main/.eslintrc.cjs